dev-util/bats/bats-1.3.0.ebuild

```diff
@@ -13,12 +13,21 @@ SRC_URI="https://github.com/${MY_PN}/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="amd64 ~arm ~arm64 ppc64 ~riscv x86"
+IUSE="test"
```

Comment:
this is needed because of the `if use test` in `src_prepare` ... not sure whether this one change will need a `-r1`, so far the versions stay untouched.

Reply:
Why is the sed even done conditionaly? You could probably simply do it in src_test. Also the sed is missing a `|| die`.

Reply:
i first had it in `src_test` but since it is a "patch" i opted for `src_prepare` since that seems much cleaner ... so `src_test` just runs the tests but does not alter the sources ... which should probably be done by `src_prepare` as a general rule.

the `|| die` is a very good point which i will adress, not sure unconditional is a good idea ... will try that and wait for further reviews

Reply:
did add the `|| die` and unconditional `sed`